{{cookiecutter.project_slug}}/{{cookiecutter.project_slug}}/utils.py

```diff
@@ -1,3 +1,19 @@
+from cryptography.fernet import Fernet
+from django.conf import settings
+
+FERNET = Fernet(settings.DB_ENCRYPTION_KEY)
+
+
+def fernet_encrypt(s):
+    """Encrypt a string using cryptography.fernet"""
+    return FERNET.encrypt(s.encode("utf-8")).decode("utf-8")
+
+
+def fernet_decrypt(s):
+    """Decrypt a string using cryptography.fernet"""
+    return FERNET.decrypt(s.encode("utf-8")).decode("utf-8")
+
+
```

Comment:
We should move everything in this file to sfdo-template-helpers.

Reply:
Makes sense. @wlonk You want to start by abstracting that and making the relevant changes in MD, then we can backport over here?

Reply:
@jgerigmeyer Done. I've added a commit here to import it from the correct spot.